ansible/configs/ocp4-cluster/post_infra.yml

```diff
@@ -46,3 +46,59 @@
       - "user.info: Windows Host(s): {{ windows_remote_hosts }}"
       - "user.info: Windows Password: {{ windows_password }}"
 
+  - name: AWS Post Infrastructure - get credentials
+    when: cloud_provider is match("ec2")
+    block:
+    - name: Get Route53User credentials from stack outputs
+      when:
+      - cloudformation_out_final is defined
+      - cloudformation_out_final.stack_outputs.Route53User is defined
+      - cloudformation_out_final.stack_outputs.Route53UserAccessKey is defined
+      - cloudformation_out_final.stack_outputs.Route53UserSecretAccessKey is defined
+      set_fact:
+        route53user: "{{ cloudformation_out_final.stack_outputs.Route53User }}"
+        route53user_access_key: "{{ cloudformation_out_final.stack_outputs.Route53UserAccessKey }}"
+        route53user_secret_access_key: "{{ cloudformation_out_final.stack_outputs.Route53UserSecretAccessKey }}"
+      tags:
+      - provision_cf_template
+    - name: Get StudentUser credentials from stack outputs
+      when:
+      - cloudformation_out_final is defined
+      - cloudformation_out_final.stack_outputs.StudentUser is defined
+      - cloudformation_out_final.stack_outputs.StudentUserAccessKey is defined
+      - cloudformation_out_final.stack_outputs.StudentUserSecretAccessKey is defined
+      set_fact:
+        student_stack_user: "{{ cloudformation_out_final.stack_outputs.StudentUser }}"
+        student_access_key_id: "{{ cloudformation_out_final.stack_outputs.StudentUserAccessKey }}"
+        student_secret_access_key: "{{ cloudformation_out_final.stack_outputs.StudentUserSecretAccessKey }}"
+      tags:
+      - provision_cf_template
+
+- name: Step 002.1 - Post Infrastructure Bastion
+  hosts: bastions
+  become: false
+  tags:
+  - step002.1
+  - post_infrastructure
+  tasks:
+  - name: AWS configuration for Bastions
+    when: cloud_provider == 'ec2'
+    block:
+    - name: Create .aws for awscli
+      file:
+        path: ~/.aws
+        mode: 0700
+        state: directory
+
+    - name: Write down Route53User credentials
```

Comment:
Do we need this in addition to the credentials that are written out later on? Do these update a different r53 zone that the sandbox or gpte creds?

Reply:
Also, just making a note like the others, this should also be removed afterwards.

Reply:
Those are for Lets Encrypt. At least they were...